pkg/codegen/python/utilities.go

```diff
@@ -185,3 +186,17 @@ func pypiVersion(v semver.Version) string {
 	}
 	return fmt.Sprintf("%d.%d.%d%s%s%s%s", v.Major, v.Minor, v.Patch, release, dev, post, local)
 }
+
+// pythonCase converts s to PascalCase, ignoring underscores, e.g. __myWords -> __MyWords.
+func pythonCase(s string) string {
+	var underscores string
+	noUnderscores := strings.TrimLeftFunc(s, func(r rune) bool {
+		if r != '_' {
+			return false
+		}
+		underscores += "_"
+		return true
+	})
+	c := cgstrings.Camel(noUnderscores)
+	return underscores + strings.ToUpper(c[:1]) + c[1:]
```

Comment:
```suggestion
	return underscores + cgstrings.Title(c)
```

Reply:
This turned into `cgstrings.UppercaseFirst(c)`.